script.js

```diff
@@ -1,4 +1,146 @@
+  if (input == "korean") {
+    gamemode = "korean";
+    return "Game mode is now Korean";
+  }
+  if (gamemode == "reverse") {
+    return winCondition(computerChoice, input);
+  }
+  if (gamemode == "normal") {
+    return winCondition(input, computerChoice);
+  }
+  if (gamemode == "korean") {
+    return koreanGame(input, computerChoice);
+  }
+};
+
+var winCondition = function (input, computerChoice) {
+  // Default lose unless fulfill win condition
+  if (input == "scissors" && computerChoice == "paper") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (input == "paper" && computerChoice == "stone") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (input == "stone" && computerChoice == "scissors") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (computerChoice == input) {
+    myOutputValue = "DRAW!!";
+  } else myOutputValue = "You lose...";
+
+  // Add emoji
+  if (computerChoice == "scissors") {
+    computerChoice = computerChoice + " ✂️";
+  } else if (computerChoice == "paper") {
+    computerChoice = computerChoice + " 🧻";
+  } else {
+    computerChoice = computerChoice + " 🪨";
+  }
+
+  if (input !== "scissors" && input !== "paper" && input !== "stone") {
+    return "Oops, please enter a valid choice (scissors / paper / stone)";
+  }
+  // add attempt count if valid input
+  attempts += 1;
+
+  if (gamemode == "normal") {
+    myOutputValue =
+      myOutputValue +
+      ` The computer used ${computerChoice}` +
+      `<br><br> You have played ${attempts} game(s) and won ${wins} time(s)` +
+      "<br><br> Enter (Scissors/Paper/Stone) to play another round (remember to change the gamemode if needed)!";
+    return myOutputValue;
+  } else {
+    myOutputValue =
+      myOutputValue +
+      ` The computer used ${input}` +
+      `<br><br> You have played ${attempts} game(s) and won ${wins} time(s)` +
+      "<br><br> Enter (Scissors/Paper/Stone) to play another round (remember to change the gamemode if needed)!";
+    return myOutputValue;
+  }
+};
+
+var koreanGame = function (input, computerChoice) {
+  if (input == "scissors" && computerChoice == "paper") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (input == "paper" && computerChoice == "stone") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (input == "stone" && computerChoice == "scissors") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (computerChoice == input) {
+    myOutputValue = "DRAW!!";
+  } else myOutputValue = "You lose...";
+
+  console.log(input);
+  console.log(computerChoice);
+  console.log(myOutputValue);
+  console.log(currentWinner);
+  if (myOutputValue == "You win!") {
+    currentWinner = "you";
+    return (
+      "MUK-JJI-PPA!! <br><br>" +
+      `You used ${input} and computer used ${computerChoice}` +
+      "<br><br> Quickly enter another choice of either scissors, paper or stone! <br><br>" +
+      `Current Winner: ${currentWinner}`
+    );
+  }
```

Comment:
This is a unique way of determining the steps that follow after the winner has been found.